Fixes 38 Surplus being unprintable in the autolathe and 38 speedloader description

Gristlebee opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Fixes #2814 and makes 38 surplus ammo able to be printed from the autolathe. Changes the damage blurb on the 38 speedloader to reflect standard 38 ammo's new stats.
## Why It's Good For The Game
Bug fixes are good.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
fix: 38 Surplus ammo should be printable in the autolathe.
fix: 38 speedloader damage blurb is accurate to it's new stats.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Gristlebee committed:
Fixes 38 Surplus being unprintable

west3436 commented on code/modules/research/designs/autolathe_designs.dm:
```diff
@@ -798,10 +798,10 @@
+/datum/design/c38_surplus
 	name = "Ammo Box (.38 surplus)"
-	id = "c38"
-	build_type = AUTOLATHE | PROTOLATHE
```
Stealth prolathe nerf - add this to the PR description

Gristlebee replied:
Added.

I'll also note #2775 already touched on this and restricted 38 ammo to the security protolathe only. No protolathe is also currently able to print surplus ammo either.

Gristlebee replied:
Turns out Im insane and missed the fact the other surplus ammo types also have protolathe print flags. Readded the flag for consistency.

Gristlebee committed:
Merge branch 'master' into Surplus-38-Ammo-fix

Gristlebee committed:
Readds protolathe flag to surplus 38 design, changes 38 speedloader desc

Standardizing the designs, desc reflects new ammo stats

Gristlebee commented:
Updated the speedloader desc while I was at it since 38 now has the same stats as 9mm. Changed the PR description to reflect that I added back the protolathe flag. Everything should be in order.

Gristlebee committed:
Merge branch 'master' into Surplus-38-Ammo-fix

MarkSuckerberg reviewed (approved):
good catch

Gristlebee committed:
Merge branch 'master' into Surplus-38-Ammo-fix

Imaginos16 commented:
Insane
